src/Components/Main.js

```diff
@@ -0,0 +1,16 @@
+import { Route, Switch } from 'react-router-dom';
+import Display from './Display/Display';
+import Nav from './Nav/Nav';
+import Notfound from './Notfound/Notfound';
+
+export default function Main() {
+  return (
+    <>
+      <Nav />
+      <Switch>
+        <Route path="/display/:r/:g/:b" component={Display}/>
+        <Route exact path="*" component={Notfound} />
```

Comment:
As a user, it is confusing to see "sorry, something went wrong" on page load, when everything is working. I would add a home route '/' to prevent this.